stack/tools/src/main/java/org/apache/usergrid/tools/DuplicateOrgRepair.java

```diff
@@ -529,10 +529,22 @@ public void call(Subscriber<? super OrgUser> subscriber) {
         
         @Override
         public void removeOrg(Org keeper, Org duplicate) throws Exception {
-            // we don't have a remove org API so rename org so that it is no longer a duplicate
+            
+            // rename org so that it is no longer a duplicate
             OrganizationInfo orgInfo = managementService.getOrganizationByUuid( duplicate.getId() );
             orgInfo.setName( "dup_" + keeper.getId() + "_" + RandomStringUtils.randomAlphanumeric(10) );
             managementService.updateOrganization( orgInfo );
+
+            // fix the org name index
+            OrganizationInfo orgInfoKeeper = managementService.getOrganizationByUuid( duplicate.getId() );
+            try {
+                managementService.updateOrganizationUniqueIndex( orgInfo, duplicate.getId() );
```

Comment:
this should take in the keeper org's OrganizationInfo, and the duplicate's ID.  Looks like we're passing in the duplicates OrganizationInfo as well as the ID.